<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FInHouseGuest</class>
 <widget class="QWidget" name="FInHouseGuest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="EQCheckBox" name="chDisplayRate">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Display rate</string>
     </property>
     <property name="Field" stdset="0">
      <string>r.f_pricePerNight</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQCheckBox" name="chDisplayCardex">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Display cardex</string>
     </property>
     <property name="Field" stdset="0">
      <string>cx.f_name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQCheckBox" name="chDisplayName">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Display name</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="Field" stdset="0">
      <string>concat(g.f_title, ' ', g.f_firstName, ' ', g.f_lastName)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQCheckBox" name="chAll">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>All</string>
     </property>
     <property name="Field" stdset="0">
      <string>r.f_extraBedFee;r.f_mealPrice;v.f_vat</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="WCardex" name="wCardex" native="true"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>2</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="EDateEdit" name="deDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chDate">
     <property name="text">
      <string>Search by date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="teTime">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chTime">
     <property name="text">
      <string>Search by time (less)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">eqcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>WCardex</class>
   <extends>QWidget</extends>
   <header location="global">wcardex.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chDisplayRate</sender>
   <signal>clicked(bool)</signal>
   <receiver>FInHouseGuest</receiver>
   <slot>groupCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chDisplayName</sender>
   <signal>clicked(bool)</signal>
   <receiver>FInHouseGuest</receiver>
   <slot>groupCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chDisplayCardex</sender>
   <signal>clicked(bool)</signal>
   <receiver>FInHouseGuest</receiver>
   <slot>groupCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chAll</sender>
   <signal>clicked(bool)</signal>
   <receiver>FInHouseGuest</receiver>
   <slot>groupCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>groupCheckClicked(bool)</slot>
 </slots>
</ui>
